cli/waiter/data_format.py

```diff
@@ -158,6 +159,36 @@ def load_data(options):
         if not content:
             raise Exception(f'Unable to load {input_format} from {input_file}.')
 
+        context_dict = {}
+        context_provided = False
+        context_file = options.get('context_file')
+        if context_file:
+            context_provided = True
+            logging.debug(f'reading context as yaml from {context_file}')
```

Comment:
Same question as below. Is the format always yaml, and if not, can we mention "reading context as yaml or json from..."

Reply:
Fixed messaging.